src/xpk/commands/workload.py

```diff
@@ -218,6 +218,8 @@
               {machine_label}
               {autoprovisioning_args}
             priorityClassName: {args.priority}
+            hostNetwork: true
```

Comment:
for my background, did you find any benefit to hostNetwork: false. McJax also using hostNetwork: true and sees good performance.

Reply:
Yes, we saw a significant improvement in step time after this change. Before this, PW was consistently 30-40% worse on large slices where McJAX step time was ~16 sec and PW was ~26 sec.